src/Microsoft.AspNet.StaticFiles/StaticFileContext.cs

```diff
@@ -218,44 +215,38 @@ private void ComputeRange()
                 return;
             }
 
-            string rangeHeader = _request.Headers.Get(Constants.Range);
-            IList<Tuple<long?, long?>> ranges;
-            if (!RangeHelpers.TryParseRanges(rangeHeader, out ranges))
+            var rangeHeader = _requestHeaders.Range;
+            if (rangeHeader == null)
             {
                 return;
             }
 
-            if (ranges.Count > 1)
+            if (rangeHeader.Ranges.Count > 1)
             {
-                // multiple range headers not yet supported
-                _logger.WriteWarning("Multiple range headers not yet supported, {0} ranges in header", ranges.Count.ToString());
+                _logger.WriteWarning("Multiple ranges not yet supported: '{0}'", rangeHeader.ToString());
```

Comment:
What exactly does this mean? Is this a feature we haven't built yet? If so, it should be an exception and explode, and we should have a bug tracking it. It shouldn't just silently fail, right?